codec-http/src/main/java/io/netty/handler/codec/http/multipart/MixedAttribute.java

```diff
@@ -271,7 +271,7 @@ public String getValue() throws IOException {
     @Override
     public void setValue(String value) throws IOException {
         if (value != null) {
-            checkSize(value.getBytes().length);
+            checkSize(value.getBytes(getCharset()).length);
         }
```

Comment:
Looking at this one… both `DiskAttribute` and `MemoryAttribute` already do this check themselves, so I wonder if we can just remove the check here and let the delegate take care of it. Then we won't compute the byte array twice.

Reply:
Will have a look to it to check

Reply:
You're right! I change it

Reply:
Done